sonic_data_client/transl_data_client.go

```diff
@@ -131,7 +131,8 @@ func (c *TranslClient) StreamRun(q *queue.PriorityQueue, stop chan struct{}, w *
 	for i,sub := range subscribe.Subscription {
 		stringPaths[i] = c.path2URI[sub.Path]
 	}
-	subSupport,_ := translib.IsSubscribeSupported(stringPaths)
```

Comment:
why is this part of the build steps and so does db_dump file for test?

Reply:
It was included due to a merge with master not working as expected. I have remerged and these changes no longer show up. All of the "changes" in files other than the readme.md were actually already in master.